### ISSUE TITLE
pygame.error:Unable to open file 'a.wav'

### ISSUE THREAD

**Issue by joglea** [raspberrypilearning/gpio-music-box#16]
my musicbox.py code:

```python
import pygame.mixer
from pygame.mixer import Sound
from gpiozero import Button
from signal import pause

pygame.mixer.init()

sound_pins = {
    2: Sound("a.wav"),
}

buttons = [Button(pin) for pin in sound_pins]
for button in buttons:
    sound = sound_pins[button.pin]
    button.when_pressed = sound.play

pause()
```

 it throws this error:

```
Traceback (most recent call last):
  File "/home/pi/musicbox/musicbox.py", line 9, in <module>
    2: Sound("a.wav"),
pygame.error: Unable to open file 'a.wav'
```

The `a.wav` file is saved in the same directory as the python file the code is in.

I can't understand why it won't work!

**Comment by bennuttall** [comment]
Thanks for letting us know.

*Note: I added fenced code blocks around your code to make it more readable. Use 3 backticks to start and stop.*

I have just tested this with a wav file I downloaded from the internet and it worked as expected.

However, I tried to replicate what the worksheet says (using wav files from Sonic Pi) and found these are now all .flac files not .wav. Trying to create a Sound object with one of these .flac files produces the same error you have: `pygame.error: Unable to open file 'a.wav'`

I imagine pygame is unable to play the specific wav file you're trying to play. It's probably encoded in a way it doesn't understand. Or perhaps, did you convert it yourself (or rename the file extension)?

Try downloading the sample file I used (from [here](http://www.wavsource.com/movie_stars/bogart.htm)) by entering the following in a terminal in the same directory was your code:

```bash
wget http://goo.gl/D6ztxR -O dtc.wav
```

And try playing that sound with pygame. That will indicate your wav file is incompatible.

**Comment by joglea** [comment]
Thanks ~~~
At first, I followed the [lesson](https://www.raspberrypi.org/learning/gpio-music-box/worksheet/)  step by step. I used .flac files from Sonic Pi. It throws the same error:pygame.error: Unable to open file 'xxxx.flac'.
Then I tried to used the .wav files I downloaded from freesound.org. I did not  convert it  or rename the file extension. And it  throws the same error.
Tonight  I will try your .wav file .
